CHANGELOG

```diff
@@ -1,3 +1,10 @@
+Version 2.0.2 (2020-06-19)
```

Comment:
Because we haven't released to PyPI yet, this could probably be lumped together with 2.0.1. (Which, I now realize, should probably say `Version 2.0.1 (Not yet released)` rather than `Version 2.0.1 (2020-05-28)`). And then we should update the date when we're preparing to actually release to PyPI.

Reply:
...though I don't have a great understanding of how the release process for this works, so I'm not super confident about this suggestion.

Reply:
Don't look to me for a better understanding of how the release process works. :-)  I'd be in favor of lumping them together, and of giving the date as "not yet released".

Reply:
does anyone know how the release process works? I am fine with lumping them together and since pypi only has 2.0 I think it makes sense.

Reply:
I think if anyone on the team does Robyn might. If nobody on the team does, we should probably develop some sort of understanding going forward.